etc/service-bus.api.ts

```diff
@@ -0,0 +1,411 @@
+  createSubscriptionClient(topicName: string, subscriptionName: string, options?: SubscriptionClientOptions): SubscriptionClient;
+  createTopicClient(topicName: string): TopicClient;
+  // (undocumented)
+  static getDeadLetterQueuePathForQueue(queueName: string): string;
+  name: string;
+}
+
+// @public
+interface NamespaceOptions extends NamespaceOptionsBase {
+  tokenProvider?: TokenProvider;
+}
+
+// @public
+export function parseConnectionString<T>(connectionString: string): ParsedOutput<T>;
+
+// @public (undocumented)
+class QueueClient extends Client {
+  // WARNING: The type "ConnectionContext" needs to be exported by the package (e.g. added to index.ts)
+  constructor(name: string, context: ConnectionContext, options?: QueueClientOptions);
+  // WARNING: The type "AcceptSessionOptions" needs to be exported by the package (e.g. added to index.ts)
+  // (undocumented)
+  acceptSession(options?: AcceptSessionOptions): Promise<MessageSession>;
+  cancelScheduledMessage(sequenceNumber: Long): Promise<void>;
+  cancelScheduledMessages(sequenceNumbers: Long[]): Promise<void>;
+  close(): Promise<any>;
+  // WARNING: The type "ListSessionsResponse" needs to be exported by the package (e.g. added to index.ts)
+  listMessageSessions(skip: number, top: number, lastUpdatedTime?: Date): Promise<ListSessionsResponse>;
+  peek(messageCount?: number): Promise<ReceivedMessageInfo[]>;
+  peekBySequenceNumber(fromSequenceNumber: Long, messageCount?: number): Promise<ReceivedMessageInfo[]>;
+  receive(onMessage: OnMessage, onError: OnError, options?: MessageHandlerOptions): ReceiveHandler;
+  receiveBatch(maxMessageCount: number, maxWaitTimeInSeconds?: number, maxMessageWaitTimeoutInSeconds?: number): Promise<ServiceBusMessage[]>;
+  receiveDeferredMessage(sequenceNumber: Long): Promise<ServiceBusMessage | undefined>;
+  receiveDeferredMessages(sequenceNumbers: Long[]): Promise<ServiceBusMessage[]>;
+  // WARNING: The type "SessionHandlerOptions" needs to be exported by the package (e.g. added to index.ts)
+  receiveMessgesFromSessions(onSessionMessage: OnSessionMessage, onError: OnError, options?: SessionHandlerOptions): Promise<void>;
+  receiveMode: ReceiveMode;
+  renewLock(lockTokenOrMessage: string | ServiceBusMessage): Promise<Date>;
+  scheduleMessage(message: SendableMessageInfo, scheduledEnqueueTimeUtc: Date): Promise<Long>;
+  scheduleMessages(messages: ScheduleMessage[]): Promise<Long[]>;
+  send(data: SendableMessageInfo): Promise<Delivery>;
+  sendBatch(datas: SendableMessageInfo[]): Promise<Delivery>;
+}
+
+// @public
+interface QueueClientOptions {
+  receiveMode?: ReceiveMode;
+}
+
+// @public
+interface ReceivedMessageInfo {
+}
+
+// @public
+class ReceiveHandler {
+  // WARNING: The type "MessageReceiver" needs to be exported by the package (e.g. added to index.ts)
+  // WARNING: The type "MessageReceiver" needs to be exported by the package (e.g. added to index.ts)
+  constructor(receiver: MessageReceiver);
```

Comment:
"MessageReceiver" is an internal type and I don't expect this to be used by the customer.

Reply:
As per recent changes, we no longer have a ReceiveHandler, so N/A.